pages/api/preview.js

```diff
@@ -14,6 +14,14 @@ export default async (req, res) => {
   //   return res.status(401).json({ message: 'Invalid slug' })
   // }
 
+  const corsOrigin =
+    process.env.NODE_ENV === 'development'
+      ? `http://localhost:3333`
+      : `https://pagebuilder-studio.webriq.com`
```

Comment:
Let's get this configurable via `process.env.SANITY_STUDIO_URL`

Reply:
Updated file contents and created local .env file for `SANITY_STUDIO_URL`